metadata/metadata.go

```diff
@@ -81,12 +81,12 @@ func Pairs(kv ...string) MD {
 	return md
 }
 
-// Len returns the number of items in md.
+// Len returns the number of items in MD.
```

Comment:
Godocs often refers to parameters or receivers by their name; I think this is fine.